.github/workflows/android_apk_build.yaml

```diff
@@ -0,0 +1,37 @@
+name: Build Android APK debug file
+
+on:
+  push:
+    paths:
+    - 'android/**'
+    - '.github/workflows/android_apk_build.yaml'
+
+jobs:
+  build:
+    name: Checkout and package
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Checkout to push branch
+        uses: actions/checkout@master
+        with:
+          ref: ${{ github.ref }}                                                              # checkout to branch which was pushed
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1                                                           # setup jdk environment to run gradle for compile
+        with:
+          java-version: 1.8
+      - name: Build APK file
+        if: job.status == 'success'                                                           # wait until environment is ready
+        run: |
+          cd android
+          wget https://eaterslab.herokuapp.com/api.yaml                                       # get current api specification from server
+          chmod +x ./api_generate_kotlin.sh
+          ./api_generate_kotlin.sh api.yaml
+          export GRADLE_USER_HOME=$(pwd)/.gradle
+          chmod +x ./gradlew
+          ./gradlew assembleDebug                                                             # run gradle compilation process in android project directory
+      - name: Upload
+        uses: actions/upload-artifact@v1                                                      # upload built apk as artifact tobe downloaded from workflow results
```

Comment:
How about "Put built APK in pushed branch files"?

Reply:
it's not pushed to project files, it's a part of workflow run result that can be accessed